Documentation/ApiOverview/Events/Events/Backend/AfterRawPageRowPreparedEvent.rst

```diff
@@ -0,0 +1,36 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; AfterRawPageRowPreparedEvent
+..  _AfterRawPageRowPreparedEvent:
+
+============================
+AfterRawPageRowPreparedEvent
+============================
+
+..  versionadded:: 13.3
+
+The PSR-14 event
+:php:`\TYPO3\CMS\Backend\Tree\Repository\AfterRawPageRowPreparedEvent`
+allows to modify the populated properties of a page and children records before
+the page is displayed in a page tree.
+
+This can be used to, for example, change the title of a page or apply a
+different sorting to its children.
+
+This event is called in the internal class
```

Comment:
Is this relevant and has any value to the developer implementing the event? The dispatching event can move to another class on refactoring, so this might not be true in the future. So I suggest to omit it here.

Reply:
Good point, done